### DISCUSSION TITLE
Initramfs output not on serial

### DISCUSSION THREAD

**Discussion by f-gia** [OE4T/meta-tegra#1070] [Q&A] [answered]
Hi, we are working on a Xavier NX devkit with eMMC on the kirkstone branch.

We were trying to make some changes to the init script in the initramfs, but at first we thought the initramfs step was being skipped as we saw no output of the messages echoed by the script. Then we tried launching a shell inside the script with `exec sh`, as a result the boot process stops as expected, but there is no prompt and no way to interact with the newly launched shell. So, we think the standard input/output of the initramfs is not bound to the serial console. We did not change the kernel commandline nor the dts. Any ideas why this might be happening?

**Comment by madisongh** [comment, marked as answer]
The default kernel command line has two `console=` settings, one for the serial console, and one for the video console (duplicating the default settings in L4T/JetPack). By default, the latter gets used.   Either reversing the order, or just removing the entry for the video console, should fix this for you.
